apps/docs/pages/guides/platform/migrating-and-upgrading-projects.mdx

```diff
@@ -132,6 +132,10 @@ const NEW_PROJECT_SERVICE_KEY = 'new-project-service-key-yyy'
 })()
 ```
 
+### Transfer to a different organization
+
+Note that project migration is for transferring your projects to different regions. If you need to move your project to a different organization without touching the infrastrusture, see [project transfers](https://supabase.com/docs/guides/platform/project-transfer).
```

Comment:
```suggestion
Note that project migration is for transferring your projects to different regions. If you need to move your project to a different organization without touching the infrastrusture, see [project transfers](/docs/guides/platform/project-transfer).
```

When using relative links, they also work on local/previews and respect the current url 👍